classes/class-handler.php

```diff
@@ -331,6 +331,10 @@ private function get_default_settings() {
 					'imgur',
 				],
 			],
+			'unregisterFormatType' => [
+				'core/text-color',
+				'core/image',
+			],
```

Comment:
Open to better names for this option too, I just named it the same as the G function it was calling, there's probably a better naming format..